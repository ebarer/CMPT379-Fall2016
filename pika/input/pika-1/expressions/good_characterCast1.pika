exec {
	const c := ^x^.
	const n := 117.
	const d := 128*n + [c | int].
	print c; d, _n_.
	const ctwo := [d | char].
	print ctwo, _n_.
}