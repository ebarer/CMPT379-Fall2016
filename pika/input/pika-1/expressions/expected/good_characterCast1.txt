x 15096
x
